### PR TITLE
adding projects : projects update

### PR DESCRIPTION
**Pull request by adithyakam**
Hi ,

Iam adding my 3 projects .please check and accept the pull request.
any suggestions are welcomed.

thanks in advance

### CODE REVIEW

File: content/participants/ttjc@adithyakam/index.md [hunk 1]
```diff
@@ -38,22 +38,36 @@ Can't Wait to fill this area!!! Stay tuned
 
 My projects:
 
-<!--
-#### binder: tinder for books
+#### Moviesss
 
-_description_ do you wish to meet people who have read the same book so that you can talk hours about the same book? Try this.
+_description_ : It is a kind of netflix clone,where user canserch for movies,trendingmovies and also add movies to there favorities.
```
Comment (khusharth) [inline content/participants/ttjc@adithyakam/index.md:43]: Hey! @adithyakam some small mistakes here:
* where user **can search** for movies, **trending movies** and also add movies to **their favorites**.

File: content/participants/ttjc@adithyakam/index.md [hunk 2]
```diff
@@ -38,22 +38,36 @@ Can't Wait to fill this area!!! Stay tuned
...
+_hosted link_ : https://moviesss.netlify.app/
 
-_github link_ https://github.com/tanaypratap/binder
+_github link_ : https://github.com/adithyakam/moviessss
 
-#### another project
+#### Chatzz
 
-_description_
+_description_ : do you want to have a simple conversations.This app is used to chat with users,to start chatting please login by using an display name and chat room you want to enter.
```
Comment (khusharth) [inline content/participants/ttjc@adithyakam/index.md:53]: * conversation*

File: content/participants/ttjc@adithyakam/index.md [hunk 3]
```diff
@@ -38,22 +38,36 @@ Can't Wait to fill this area!!! Stay tuned
...
-_stack_ -->
+_stack_ : It is made with React with materializeCSS .It uses Socket.IO for socket connection and express as backend.
+
+_hosted link_ : https://chatzz.netlify.app/
+
+_github link_ : https://github.com/adithyakam/chat-app
+
+#### Expense Traccker
+
+_description_ : This app is used tomanage personal finance .App gives way of spending in chart for montly categorywise.
```
Comment (khusharth) [inline content/participants/ttjc@adithyakam/index.md:63]: * is used **to manage** personal 
* category wise

File: content/participants/ttjc@adithyakam/index.md [hunk 4]
```diff
@@ -38,22 +38,36 @@ Can't Wait to fill this area!!! Stay tuned
...
+additionalNote: you can use a@gmail.com and 1234 for login
+
+_stack_ : It is made with React with React-Router.it uses Express as backend with Rest API,and Postgress as database.
```
Comment (khusharth) [inline content/participants/ttjc@adithyakam/index.md:66]: * It* uses Express

**Comment by adithyakam** [comment]
thanks for the review ,
All the mistakes has been updated please accept the PR

**Comment by khusharth** [comment]
@gkanishk  Good to merge.

**Pull request merged**